docs/docs/quick_start/advanced_features/node_expansion.md

```diff
@@ -4,3 +4,79 @@ sidebar_label: "Node Expansion"
 ---
 
 # Node Expansion
+
+It's one of common requirements for a storage system to expand the capacity by adding a new storage node. HwameiStor provides this feature by the following simple procedure.
```

Comment:
```suggestion
A storage system is ususally expected to expand its capacity by adding a new storage node. In HwameiStor, it can be done with the following steps.
```

Reply:
better :) fixed